docs/_includes/permalinks.html

```diff
@@ -0,0 +1,12 @@
+<script>
```

Comment:
This change seems to have affected the automatic creation and listing of the per page table of contents in the right hand side nav menu.

**Before**

<img width="1222" alt="Screen Shot 2020-05-20 at 6 35 35 AM" src="https://user-images.githubusercontent.com/7570704/82442742-d9857d00-9a65-11ea-954e-86355582fc37.png">

**After**

<img width="1219" alt="Screen Shot 2020-05-20 at 6 35 27 AM" src="https://user-images.githubusercontent.com/7570704/82442746-ddb19a80-9a65-11ea-9d45-a4906979e0a2.png">

Reply:
Are you sure? Because I added this spacing because this table was not showing like this in your ss. Columns were overlapping and also no CSS was showing.

Reply:
FWIW, I'm using Safari to look at the before and after changes.

Reply:
Fixed!